WcaOnRails/public/images/organizations/swisscubing.svg

```diff
@@ -0,0 +1,248 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+  "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg width="336" height="334">
+  <image id="image0" width="336" height="334" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAVAAAAFOCAYAAADQNEUyAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```

Comment:
This one:

![swisscubing](https://cloud.githubusercontent.com/assets/277474/22176306/1309c620-dfbc-11e6-8674-32c7e890ee74.png)